bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT_LO_MLM/tt0123j_5f_ckm_HT-2500toInf_LO_MLM/tt0123j_5f_ckm_HT-2500toInf_LO_MLM_madspin_card.dat

```diff
@@ -0,0 +1,36 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+# 
+#to properly limit the number of concurrent processes for grid running
+set max_running_process 1
+#
+# specify the decay for the final state particles
+ decay t > w+ b, w+ > all all
+ decay t~ > w- b~, w- > all all
+ decay w+ > all all
+ decay w- > all all
+
+#define ell+ = e+ mu+ ta+
```

Comment:
can you please check all the commented definitions...

Reply:
Hi Luca, what do you mean by check the definitions? You mean check whether they need to be uncommented?

Reply:
no, I mean that ALL the definitions that are commented should be deleted

Reply:
Okay thank you for the clarification, it should be done now